CHANGES.md

```diff
@@ -111,6 +111,7 @@ To be released.
     [[#1129]]
  -  Store aliases used by `planet mpt` became to disallow names looking like
     URIs to disambiguate aliases from the literal store URIs.  [[#1129]]
+ -  Added a new sub-command `planet mpt list` and `planet mpt query`.  [[#1137]]
```

Comment:
```suggestion
 -  Added new subcommands `planet mpt list` and `planet mpt query`.  [[#1137]]
```

FYI English speakers seems to prefer “subcommand” without a hyphen ([about 787,000 results on Google][1]) over “sub-command” with a hyphen ([about 154,000 results on Google][2]).

[1]: https://www.google.com/search?q=cli+%22subcommand%22
[2]: https://www.google.com/search?q=cli+%22sub-command%22

Reply:
I amended it in adefc27.